src/styles/reset.css

```
@@ -0,0 +1,43 @@
+}
+
+html {
+    min-height: 100vh;
+}
+
+body {
+    margin: 0;
+    padding: 0;
+    font-size: 18px;
+    line-height: 1.5;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Inter', sans-serif;
```

Stylelint (17.11.1)

[error] Expected no quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)